control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -340,6 +340,10 @@ func buildCVOContainerMain(image string) func(c *corev1.Container) {
 			},
 		}
 		c.VolumeMounts = volumeMounts.ContainerMounts(c.Name)
+		c.Resources.Requests = corev1.ResourceList{
```

Comment:
What is the basis for all these new values? We have a CI verification btw that ensures that everything has resource requests set. I don't know offhand where they come from on every component, but they are set. The fact that you are adding them rather than changing means that this change is  incorrect.